src/overlays/actors/ovl_Bg_Toki_Swd/z_bg_toki_swd.c

```diff
@@ -1,14 +1,306 @@
+/*
+ * File: z_bg_toki_swd.c
+ * Overlay: ovl_Bg_Toki_Swd
+ * Description: Master Sword (Contains Cutscenes)
+*/
+
 #include "z_bg_toki_swd.h"
 
 #define ROOM  0x00
 #define FLAGS 0x00000010
 
-void BgTokiSwd_Init(BgTokiSwd* this, GlobalContext* globalCtx);
-void BgTokiSwd_Destroy(BgTokiSwd* this, GlobalContext* globalCtx);
-void BgTokiSwd_Update(BgTokiSwd* this, GlobalContext* globalCtx);
-void BgTokiSwd_Draw(BgTokiSwd* this, GlobalContext* globalCtx);
+static void BgTokiSwd_Init(BgTokiSwd* this, GlobalContext* globalCtx);
+static void BgTokiSwd_Destroy(BgTokiSwd* this, GlobalContext* globalCtx);
+static void BgTokiSwd_Update(BgTokiSwd* this, GlobalContext* globalCtx);
+static void BgTokiSwd_Draw(BgTokiSwd* this, GlobalContext* globalCtx);
+static void BgTokiSwd_SetupAction(BgTokiSwd* this, ActorFunc actionFunc);
+static void func_808BAF40(BgTokiSwd* this, GlobalContext* globalCtx);
+static void func_808BB0AC(BgTokiSwd* this, GlobalContext* globalCtx);
+static void func_808BB128(BgTokiSwd* this, GlobalContext* globalCtx);
+
+extern u32 D_06001BD0;
+
+static u32 D_808BB2F0[] =
+{
+    0x00000009, 0x000001A9, 0x0000000A, 0x00000001, 
```

Comment:
We should probably make raw data like this take up as few lines as possible unless it's organized like this for a reason. I'd make it so you can have as many on a line as possible without crossing our 120 char limit...but maybe don't do this because clang format / clang tidy will fix it? @Roman971 thoughts?

Reply:
Don't bother manually formatting it for sure, clang-format will handle it. But this amount of data belongs in its own file imo.